### PR TITLE
Python for Networking Devices Article cont.

### PR DESCRIPTION
**Pull request by fooelisa**
adding summary, correcting a few more comments from PR #47

### CODE REVIEW

**Comment by deshipu** [comment]
Looks great.

File: 2016/presentations/Python_for_Networking_Devices/text.md [hunk 1]
```diff
@@ -102,28 +103,37 @@ libraries to facilitate device interaction - to mention a few:
...
+used for each of them individually, however managing the use of different libraries per vendor is quite painful.
 
-Efforts to improve this have been started, for example with [netmiko](https://github.com/ktbyers/netmiko) which is a
+In an effort to improve upon this situation, wrappers around a set of libraries have been built to unify multivendor
+access to networking devices. For example [netmiko](https://github.com/ktbyers/netmiko), which is a
 wrapper around paramiko and provides easy SSH access for plenty of networking equipment. On the Netconf front, or to
 be more specific, access which supports transactions on the device, a project called
-[napalm](https://github.com/napalm-automation/napalm) has been started last year.
+[napalm](https://github.com/napalm-automation/napalm) has been started, which unifies a set of vendor libraries
+into a single set of methdos.
```
Comment (orbitvu) [inline 2016/presentations/Python_for_Networking_Devices/text.md]: s/methdos/methods

Comment (pigletto) [inline 2016/presentations/Python_for_Networking_Devices/text.md]: s/methdos/methods

File: 2016/presentations/Python_for_Networking_Devices/text.md [hunk 2]
```diff
@@ -102,28 +103,37 @@ libraries to facilitate device interaction - to mention a few:
...
-interworkings between different vendors have been proven to take a lot of time and effort and not be on the market as
-quickly as the engineers would like.
+Of course there are also standardization efforts, like open config or Netconf, which aim to improve upon this
+situation. But as far as standards go, interworkings between different vendors have been proven to take a lot of time
+and effort, and typically aren't on the market as quickly as the engineers would like.
 
-##The End
+##Summary
+
+How to start if you are looking at intoducing automation into your network? Start by reviewing your network
```
Comment (pigletto) [inline 2016/presentations/Python_for_Networking_Devices/text.md]: s/intoducing/introducint

**Comment by fooelisa** [comment]
Fixed a few more typos, thanks everyone! This is good to close

**Comment by nhlfr** [comment]
LGTM

**Pull request merged**